go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	k8s.io/component-base v0.27.2
 	k8s.io/kubectl v0.27.2
 	k8s.io/kubelet v0.27.2
+	k8s.io/kubernetes v1.27.3
```

Comment:
not mandatory but for consistency and avoid any surprise maybe we should also update other kube deps to match 1.27.3

Reply:
done!